spaces/domains.json

```
@@ -723,5 +723,6 @@
   "gov.thaloz.com": "thaloz.eth",
   "vote.aura.finance": "aurafinance.eth",
   "vote.rocketpool.net": "rocketpool-dao.eth",
-  "vote.metapocket.games": "meta-pocket.eth"
+  "vote.metapocket.games": "meta-pocket.eth",
+  "vote.cortexdao.io": "cortexdao.eth",
```

Please remove the comma at the end of the line.
```suggestion
  "vote.cortexdao.io": "cortexdao.eth"
```